### PR TITLE
WIP: Document cfd trading flow

### PR DESCRIPTION
**Pull request by holzeis**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -24,3 +24,7 @@
 /testnet
 !/testnet/maker_seed
 !/testnet/taker_seed
+
+
+# Clion IDE
+.idea
```
Comment (bonomat) [inline .gitignore]: Happy to leave this here. You can also exclude local files without changing `.gitignore` if you add them into  `.git/info/exclude`

File: docs/asset/cfd_trading.puml [hunk 2]
```diff
@@ -0,0 +1,50 @@
+@startuml
```
Comment (bonomat) [inline docs/asset/cfd_trading.puml:1]: Can we move this file into a different repository? It includes information out of scope of this project e.g. `maker_automation`. I propose to move it into `meta`.

File: docs/asset/cfd_trading.puml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+participant MakerAutomation
+participant BitMex
+
+
+== Offer Protocol ==
+TakerDaemon->MakerDaemon: Connect
+|||
+loop heartbeat
+|||
+    MakerDaemon->BitMex: Check bid & ask prices
```
Comment (bonomat) [inline docs/asset/cfd_trading.puml:14]: Not 100% correct: we subscribe to the `quoteBin1m` topic, meaning, we get unconditionally a new quote every minute.

File: docs/asset/cfd_trading.puml (hunk 2 above)
Comment (holzeis) [reply docs/asset/cfd_trading.puml:1]: fine with me.. I will move the pull request then.

**Comment by holzeis** [comment]
moved this pull request to the meta project - https://github.com/itchysats/meta/pull/57. closing this one.

**Pull request closed without merging**